virt/vm_networking/virt-configuring-viewing-ips-for-vms.adoc

```diff
@@ -0,0 +1,41 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-viewing-ips-for-vms"]
+= Configuring and viewing IP addresses
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-viewing-ips-for-vms
+
+toc::[]
+
+You can configure an IP address when you create a virtual machine (VM). The IP address is provisioned with cloud-init.
+
+You can view the IP address of a VM by using the {product-title} web console or the command line. The network information is collected by the QEMU guest agent.
+
+[id="configuring-ips_virt-configuring-viewing-ips-for-vms"]
+== Configuring IP addresses for virtual machines
+
+You can configure a static IP address when you create a virtual machine (VM) by using the web console.
```

Comment:
Seeing as you are commenting out the respective module, does it make sense to remove line 16, as well?

If you decide to leave it in, I suggest consolidating both sentences into something like, "You can configure a static IP address when you create a virtual machine (VM) by using either the web console or command line."